### PR TITLE
383 image principale et secondaire articles

### PR DESCRIPTION
**Pull request by hugolopespinto**
- [x] Button to change images
- [x] Indicators for number of images + image actually displayed
- [x] Multiple images handling
- [x]  UT

demo:
[Screen_recording_20240721_155215.webm](https://github.com/user-attachments/assets/06b56450-281e-48f0-b6ed-9d55d805deb7)

Tests:
image

### CODE REVIEW

File: Application/mobile/app_code/lib/l10n/app_en.arb [hunk 1]
```diff
@@ -149,7 +149,6 @@
   "notificationsPreferencesManagement": "Notifications preferences management",
   "notificationsSaved": "Saved notifications preferences",
   "ok": "OK",
-  "opinions": "Opinions",
```
Comment (invpuppet) [inline Application/mobile/app_code/lib/l10n/app_en.arb:152]: Why delete this ?

File: Application/mobile/app_code/lib/l10n/app_fr.arb [hunk 2]
```diff
@@ -149,7 +149,6 @@
   "notificationsPreferencesManagement": "Gestion des préférences de notifications",
   "notificationsSaved": "Préférences de notifications enregistrées",
   "ok": "OK",
-  "opinions": "Avis",
```
Comment (invpuppet) [inline Application/mobile/app_code/lib/l10n/app_fr.arb:152]: Why delete this ?

**Review by invpuppet — CHANGES_REQUESTED** [review]

File: Application/mobile/app_code/lib/l10n/app_en.arb (hunk 1 above)
Comment (hugolopespinto) [reply Application/mobile/app_code/lib/l10n/app_en.arb]: i added it again

File: Application/mobile/app_code/lib/l10n/app_fr.arb (hunk 2 above)
Comment (hugolopespinto) [reply Application/mobile/app_code/lib/l10n/app_fr.arb]: i added it again

**Review by CedricCORGE — APPROVED** [review]

**Review by invpuppet — APPROVED** [review]

**Review by tbellicha — CHANGES_REQUESTED** [review]
TU Front Mobile

**Pull request closed without merging**